app/src/test/java/mcs/salazar/jesus/mcsbattleship/BattleshipSetupTest.java

```diff
@@ -0,0 +1,172 @@
+package mcs.salazar.jesus.mcsbattleship;
+
+import org.junit.Test;
+
+import java.util.List;
+import java.util.Random;
+
+import mcs.salazar.jesus.mcsbattleship.models.Coordinate;
+
+import static org.junit.Assert.*;
+
+/**
+ * Created by Ivan on 2/10/2018
+ */
+public class BattleshipSetupTest {
+
+    private final boolean[][] mGrid = {{false, false, false, false, false},
```

Comment:
This is not formatted, also, you may (not required, just FYI) have this in a different method . ( . getMGridTestCase1():mGrid[][]  )

private boolean[][] getMock01(){
    return ...
}